docs/2_6_versioning_layered_standards.adoc

```diff
@@ -2,19 +2,26 @@
-A layered standard defines extensions to the base FMI standard by specifying either standardized annotations, standardized extra files in the FMU, and/or support for additional MIME types/file formats, defined in <<definition-of-types>>.  +
-A layered standard can include a single or combined set of extension mechanisms from this set. +
-The layered standard is thus considered to be layered on top of the definitions and extensions mechanisms provided by this base standard.
+In order to enable the backward-compatible extension of the FMI standard in minor releases and between minor releases, the FMI project intends the use of the layered standard mechanism to introduce new features in a fully backward-compatible and optional way.
 
-Layered standards can fall into three categories:
+A layered standard defines extensions to the base FMI standard by specifying, for example:
 
-* Layered standards defined by third parties, without any representations by the FMI project for their suitability or content, or even knowledge by the FMI project about their existence.
-* Layered standards defined by third parties that are endorsed by the FMI project and listed on the FMI project website.
-* Layered standards can be defined/adopted and published by the FMI project itself, making them FMI project layered standards.
+* <<Annotations,standardized annotations>>,
+* <<extra-directory,standardized extra files in the FMU>>,
+* support for additional <<mimeType,MIME types>>,
+* new values for <<matchingRule>> or <<terminalKind>> for <<Terminals,Terminals>> and/or
+* adding C-API functions/variables.
 
-Layered standards that have achieved enough adoption or importance to be included into the base standard set could be incorporated into a new minor or major release version of the base standard as an optional or mandatory appendix, making support for this layered standard optional or required for conformance with the newly published minor release version of the base standard.
+Any FMU following a layered standard on top of FMI 3.0 must still be a compliant FMI 3.0 FMU.
+
+Layered standards can fall into one of three categories:
+
+* defined by third parties, completely independent from the <<MAP>> FMI,
+* defined by third parties that are endorsed by the FMI project and listed on the FMI project website, and
+* defined/adopted and published by the <<MAP>> FMI project itself, making them FMI project layered standards.
+
+Layered standards that have achieved enough adoption or importance to be included into the base standard could be incorporated into a new minor or major release version of the base standard as an optional or mandatory appendix, making support for this layered standard optional or required for conformance with the newly published release of the FMI standard.
```

Comment:
There is also the possibility that we move content of a layered standard in the main text of a future minor or majorr release. Is this clear or do we have to state this explicitely?

Reply:
Isn´t that what line 27 says?

Reply:
It only talks about an appendix, not about the main standard text

Reply:
I see, fixed.